homeassistant/components/litejet/config_flow.py

```diff
@@ -0,0 +1,48 @@
+"""Config flow for the LiteJet lighting system."""
+import logging
+from typing import Any, Dict, Optional
+
+import pylitejet
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PORT
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class LiteJetConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """LiteJet config flow."""
+
+    async def async_step_user(
+        self, user_input: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, Any]:
+        """Create a LiteJet config entry based upon user input."""
+        if self.hass.config_entries.async_entries(DOMAIN):
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
See previous comment about the possibility of multiple devices

Reply:
To future proof I'm putting the config entry ID (which will be based on the port value) into all of the unique_ids, but I don't plan to remove this restriction in this PR. A home with two LiteJet systems would be incredibly unusual and as I said in another comment allowing multiple would mess up the existing trigger functionality which is purely based on the ATTR_NUMBER.